test/e2e/storage/drivers/in_tree.go

```diff
@@ -163,47 +166,39 @@ func startExternalProvisioner(c clientset.Interface, ns string, externalPluginNa
 			Containers: []v1.Container{
 				{
 					Name:  "nfs-provisioner",
-					Image: "quay.io/kubernetes_incubator/nfs-provisioner:v1.0.9",
+					Image: "quay.io/external_storage/nfs-client-provisioner:latest",
```

Comment:
I think it would be better to use a specific version for the Kubernetes CI.

Reply:
@wongma7 

Which version do you recommend to use here? [v3.1.0-k8s1.11](https://quay.io/repository/external_storage/nfs-client-provisioner?tag=latest&tab=tags)?

Reply:
@msau42 

Changed the tag from latest to v3.1.0-k8s1.11, before getting answer from @wongma7 .
Because it should be better than latest and we can change it later if we find issue in v3.1.0-k8s1.11.